src/content/pages/photos.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Photos
+subtitle: EuroPython 2024 in Pictures
+---
+
+# Official Photos
+Check out the official photos from EuroPython 2024! - Enjoy and relive the memories and highlights of the week of Python in Prague & Remote - and share them with your friends and colleagues.
```

Comment:
```suggestion
Check out the official photos from EuroPython 2024! Relive the highlights of the week of Python in Prague & Remote. Don’t forget to share these memories with your friends and colleagues!
```

1. If you want to use -, you should drop the exclamation mark. 

2. There are a lot of "and"s here.  Edited for readability.